<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">QWidget {
	color: white;
	background-color: #121212;
	font-family: Rubik;
	font-size: 10pt;
	font-weight: 600;
}

QPushButton {
	background-color: transparent;
	border: 1px solid #49423d;
}

QPushButton:hover {
	background-color: #666;
}

QPushButton:!hover {
	background-color: transparent;
}

QPushButton:pressed {
	background-color: #24b3a7;
}
</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>115</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Карта Карно</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="Carnot">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>280</width>
      <height>130</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>340</y>
      <width>240</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Минимизированная СДНФ:</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="MinPDNF">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>370</y>
      <width>860</width>
      <height>60</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>450</y>
      <width>240</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Минимизированная СКНФ:</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="MinPCNF">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>480</y>
      <width>860</width>
      <height>60</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QTextEdit" name="SchaefBasis">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>40</y>
      <width>500</width>
      <height>60</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QTextEdit" name="PierBasis">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>170</y>
      <width>500</width>
      <height>60</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>10</y>
      <width>310</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Разложение по базису Шеффера:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>140</y>
      <width>260</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Разложение по базису Пирса:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Copy_MIN_PDNF">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>430</y>
      <width>120</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Копировать</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Copy_MIN_PCNF">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>540</y>
      <width>120</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Копировать</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
